sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -434,14 +434,24 @@ try {
 }
 ```
 ### When to use 'ServiceBusProcessorClient'.
- When to use 'ServiceBusProcessorClient', 'ServiceBusReceiverClient' or ServiceBusReceiverAsyncClient? The processor 
- is built using 'ServiceBusReceiverAsyncClient', it provides a convenient way of receiving messages with default 
- auto complete and auto-renew of message locks in 'PEEK_LOCK' mode. The processor is appropriate where the 
- applications have not made complete move to async receiver client and want to process message in synchronous mode. 
- The processor receives messages forever because it recovers from the network errors internally. 
- 'ServiceBusProcessorClient:processMessage()' function call is made for each message. Alternatively, You can also use 
- 'ServiceBusReceiverClient', it is a lower level client and provides a wider range of APIs. If async processing is  
- suitable for your application, you can use 'ServiceBusReceiverAsyncClient'. 
+
+When to use 'ServiceBusProcessorClient', 'ServiceBusReceiverClient' or ServiceBusReceiverAsyncClient? 
+ 
+For almost all common use cases, 'ServiceBusProcessorClient' should be the primary choice for consuming messages. The processor is built with 
+necessary resiliency to recover from errors and designed to receive messages forever. The processor offers simplified API to configure 
+concurrency for asynchronous processing.
```

Comment:
nit: 
```suggestion
concurrency.
```

My concern is only around `asynchronous`  word (since processing of each message is still sync, just parallel)

Reply:
yes, you're right, I'll change it to "The processor has a simple API to set concurrency for processing messages in parallel."